src/app/mail/mail-settings/addresses-signature/addresses-signature.component.ts

```diff
@@ -13,6 +13,50 @@ import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
 import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { Subject } from 'rxjs/internal/Subject';
 
+import Quill from 'quill';
+import ImageResize from 'quill-image-resize-module';
+
+
+// Register quill modules and fonts and image parameters
+Quill.register('modules/imageResize', ImageResize);
```

Comment:
move the shared code into shared service  and only reqister modules in this component
Only follow code needs to be in this component I think, the rest can be used in different places in app.
```
Quill.register('modules/imageResize', ImageResize);
Quill.register(ImageFormat, true);
```

Reply:
yes, Okay so now I'll move it to shared service